README.md

```diff
@@ -149,7 +174,15 @@ The ancestors have provided an example of one route; you do the other six!
 GET '/warriors/:id'
   * This is the show route, which finds a warrior by ID, and displays information about that warrior.
 
-Replace this with your answer
+POST '/warrior/:honor'
+  * this creates the warriors honor on the warrior page
+
+  PUT '/warrior/:level'
+  * this updates the warriors level based on his improved fighting skills
+
+  DELETE '/warrior/:honor'
+  * this deletes the warriors honor when he betrays the code of the warrior
```

Comment:
This is close.  The addition of :honor and :level is interesting, but clouds your answer.  I wouldn't expect values like that for those routes.  Think CRUD, with Read supporting both individual and listing of objects.